library/inclusive-ui.css

```diff
@@ -1,8 +1,17 @@
-@import url("../utils/color.css");
-@import url("../utils/reset.css");
+@import url("./components/avatar/avatar.css");
+
+
 /* general styles for components */
+:root {
+  --PRIMARY: var(--BASE);
+  --black: #171321;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-size: 1.6rem;
+}
```

Comment:
I think this can fit in your `utility` directory. Like the `html` and `body` rules can be added to `reset.css` and the `:root` can fit in `color.css`.

Reply:
yes this file will only contain the import statements from components
the main CSS file for the library
this code was just for testing purposes, I will remove this 👍